packages/loot-core/src/client/shared-listeners.ts

```diff
@@ -47,40 +49,46 @@ export function listenForSyncEvent(actions, store) {
       }
     } else if (type === 'error') {
       let notif: Notification | null = null;
-      const learnMore =
-        '[Learn more](https://actualbudget.org/docs/getting-started/sync/#debugging-sync-issues)';
+      const learnMore = i18n.t(
+        '[Learn more](https://actualbudget.org/docs/getting-started/sync/#debugging-sync-issues)',
+      );
       const githubIssueLink =
         'https://github.com/actualbudget/actual/issues/new?assignees=&labels=bug&template=bug-report.yml&title=%5BBug%5D%3A+';
 
       switch (subtype) {
         case 'out-of-sync':
           if (attemptedSyncRepair) {
             notif = {
-              title: 'Your data is still out of sync',
+              title: i18n.t('Your data is still out of sync', {}),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
              title: i18n.t('Your data is still out of sync'),
```